src/components/Sidebar/Sidebar.minors.tsx

```diff
@@ -55,6 +69,38 @@ const ItemStyled = styled(Button)<{
   }
 `;
 
+const ItemLabeledStyled = styled(Button)<{
+  isActive?: boolean;
+  activeStyles?: ActiveStyles;
+}>`
+  height: 3.25rem;
+  min-width: initial;
+  padding: 0 0.2rem;
+  margin: 0.25rem 0;
+  align-items: center;
+  flex-direction: column;
+  ${({ isActive, activeStyles }) => (isActive ? activeStyles : '')};
+
+  > svg {
+    padding: 0 !important;
+    width: 1.25rem;
+    min-width: 1.25rem;
+    max-width: 1.25rem;
+    height: 1.25rem;
+    min-height: 1.25rem;
+    max-height: 1.25rem;
+    margin: 0 0 0.45rem 0;
+  }
+
+  > span {
```

Comment:
I would add a comment here describing what this `span` is here inside the button. Ie: "Button label"